src/test/mochitest/browser_dbg-breakpoint-skipping.js

```diff
@@ -0,0 +1,29 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+add_task(async function() {
+  await pushPref("devtools.features.skip-pausing", true);
```

Comment:
@jasonLaster This isn't enabling the button within tests; is there a further update required?

Reply:
did you add the flag to `assets/panel/prefs.js`?